README.md

```diff
@@ -18,6 +18,7 @@ addon | version | summary
 [website_sale_category_megamenu](website_sale_category_megamenu/) | 8.0.1.0.1 | A megamenu to display product categories in the online shop
 [website_sale_checkout_country_vat](website_sale_checkout_country_vat/) | 8.0.1.0.0 | Autocomplete VAT in checkout process
 [website_sale_default_country](website_sale_default_country/) | 8.0.1.0.0 | Give a default value for country at checkout
+[website_sale_google_shopping](website_sale_google_shopping/) | 8.0.1.0.0 | Generate the products feed for Google Merchant Center
```

Comment:
Yes, please, don't modify this and squash the commits adding it/removing it for avoiding the said conflict